src/frontend/pages/chat/chat.js

```diff
@@ -196,11 +200,24 @@ export class Chat extends LitElement {
                           .message=${message}
                           .index=${index}
                           .activeChatName=${this.activeChatName}
+                          .activeDescription=${this.activeDescription}
+                          @forward-message=${this.openForwardMenu}
+                          @go-to-chat=${this.goToChat}
                         ></il-message>`
                     )}
                   </ul>
 
-                  <il-forward-list></il-forward-list>
+                  <il-modal
+                    .styleProperties=${{
+                      background: "#083c72",
+                      color: "white",
+                      width: "400px",
```

Comment:
esporre tutte le proprietà css non mi sembra molto gestibile: meglio definire una classe in modal selezionabile da fuori